### DISCUSSION TITLE
how to add a link within a text?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2221] [Help]
Hello, 

I need to add text like: "If you need more information, [click here](url)" 

How to do that in budibase?

**Comment by aptkingston** [comment]
Hi @oswaldB. Just put a paragraph and a link inside a container and configure the container to be a row with no gap.
![image](https://user-images.githubusercontent.com/9075550/127878969-4ab6f745-9045-4ba7-9eb8-63d58a3c6512.png)

**Reply by oswaldB** [reply]
This option doesn't work if your sentence is on 2 lines ;)

**Comment by shogunpurple** [comment]
Hi @oswaldB - another option is to just use an `embed` with some HTML, if that is simpler for you.

`If you need something click <a href="https://google.com">here </a>`
